server/dns-service/dnsservice-library/src/main/java/org/apache/james/dnsservice/library/netmatcher/NetMatcher.java

```diff
@@ -84,7 +83,7 @@ public NetMatcher(Collection<String> nets, DNSService dnsServer) {
 
     public NetMatcher(String commaSeparatedNets, DNSService dnsServer) {
         this.dnsServer = dnsServer;
-        List<String> nets = Splitter.on(NETS_SEPARATOR).splitToList(commaSeparatedNets);
+        List<String> nets = Arrays.asList(commaSeparatedNets.split(NETS_SEPARATOR_PATTERN));
```

Comment:
Can't we just trim the netwok string before calling splitToList to get to the same result?

Reply:
Well, I consider the use of String::split backed by a regular expression being a regression to be honset...

Reply:
How is this fix a "regression" when the original code does not implement what is widely regarded as comma-separation?

Reply:
> How is this fix a "regression" when the original code does not implement what is widely regarded as comma-separation?

The regrtession is about the use of regular expression in one place that was free of it.

Have you tried:

```
List<String> nets = Splitter.on(",")
    .trimResults()
    .splitToList(commaSeparatedNets);
```

?

This should solve your concerns, and pass your test suite, I guess... (without a regex)